### ISSUE TITLE
Error: media_pipe segmentation fault (core dumped)

### ISSUE THREAD

**Issue by G7b9** [SkyworkAI/SkyReels-A1#4]
`python inference.py --image_path /mnt/data/projects/SkyReels-A1/assets/ref_images/3.png --driving_video_path /mnt/data/projects/SkyReels-A1/assets/driving_video/2.mp4
运行报错`

在 skyreels_a1/src/media_pipe/face_landmark.py的第 3164 行，报错
 [1]    324703 segmentation fault (core dumped)  python inference.py --image_path  --driving_video_path

中间变量打印结果：
cls = <class 'skyreels_a1.src.media_pipe.face_landmark.FaceLandmarker'>
options.running_mode = VisionTaskRunningMode.IMAGE
options.result_callback  = None

`> pip list
Package                   Version            Editable project location
------------------------- ------------------ ----------------------------
absl-py                   2.1.0
aiofiles                  23.2.1
albucore                  0.0.23
albumentations            2.0.4
annotated-types           0.7.0
anyio                     4.6.2
argon2-cffi               21.3.0
argon2-cffi-bindings      21.2.0
asttokens                 2.0.5
async-lru                 2.0.4
attrs                     24.3.0
babel                     2.16.0
beautifulsoup4            4.12.3
black                     25.1.0
bleach                    6.2.0
Brotli                    1.0.9
certifi                   2025.1.31
cffi                      1.17.1
charset-normalizer        3.3.2
chumpy                    0.70
click                     8.1.8
colorama                  0.4.6
coloredlogs               15.0.1
comm                      0.2.1
contourpy                 1.3.1
cycler                    0.12.1
Cython                    3.0.12
debugpy                   1.8.11
decorator                 4.4.2
decord                    0.6.0
defusedxml                0.7.1
diffusers                 0.32.2
easydict                  1.13
einops                    0.8.1
exceptiongroup            1.2.0
executing                 0.8.3
facexlib                  0.3.0
fastapi                   0.115.8
fastjsonschema            2.20.0
ffmpy                     0.5.0
filelock                  3.17.0
filterpy                  1.4.5
flake8                    7.1.2
flake8-bugbear            24.12.12
flake8-comprehensions     3.16.0
flatbuffers               25.2.10
fonttools                 4.56.0
fsspec                    2025.2.0
future                    1.0.0
fvcore                    0.1.5.post20221221
gradio                    5.16.0
gradio_client             1.7.0
h11                       0.14.0
httpcore                  1.0.2
httpx                     0.27.0
huggingface-hub           0.28.1
humanfriendly             10.0
idna                      3.10
imageio                   2.37.0
imageio-ffmpeg            0.6.0
importlib_metadata        8.6.1
insightface               0.7.3
iopath                    0.1.10
ipykernel                 6.29.5
ipython                   8.30.0
ipywidgets                8.1.5
jax                       0.5.0
jaxlib                    0.5.0
jedi                      0.19.2
Jinja2                    3.1.5
joblib                    1.4.2
json5                     0.9.25
jsonschema                4.23.0
jsonschema-specifications 2023.7.1
jupyter                   1.1.1
jupyter_client            8.6.3
jupyter-console           6.6.3
jupyter_core              5.7.2
jupyter-events            0.10.0
jupyter-lsp               2.2.0
jupyter_server            2.14.1
jupyter_server_terminals  0.4.4
jupyterlab                4.3.4
jupyterlab-pygments       0.1.2
jupyterlab_server         2.27.3
jupyterlab_widgets        3.0.13
kiwisolver                1.4.8
lazy_loader               0.4
libcst                    1.6.0
llvmlite                  0.44.0
markdown-it-py            3.0.0
MarkupSafe                2.1.5
matplotlib                3.10.0
matplotlib-inline         0.1.6
mccabe                    0.7.0
mdurl                     0.1.2
mediapipe                 0.10.21
mistune                   2.0.4
ml_dtypes                 0.5.1
moreorless                0.4.0
moviepy                   1.0.3
mpmath                    1.3.0
mypy-extensions           1.0.0
narwhals                  1.27.1
nbclient                  0.8.0
nbconvert                 7.16.4
nbformat                  5.10.4
nest-asyncio              1.6.0
networkx                  3.4.2
notebook                  7.3.2
notebook_shim             0.2.3
numba                     0.61.0
numpy                     1.26.4
nvidia-cublas-cu12        12.1.3.1
nvidia-cuda-cupti-cu12    12.1.105
nvidia-cuda-nvrtc-cu12    12.1.105
nvidia-cuda-runtime-cu12  12.1.105
nvidia-cudnn-cu12         8.9.2.26
nvidia-cufft-cu12         11.0.2.54
nvidia-curand-cu12        10.3.2.106
nvidia-cusolver-cu12      11.4.5.107
nvidia-cusparse-cu12      12.1.0.106
nvidia-cusparselt-cu12    0.6.2
nvidia-nccl-cu12          2.19.3
nvidia-nvjitlink-cu12     12.4.127
nvidia-nvtx-cu12          12.1.105
onnx                      1.17.0
onnxruntime               1.20.1
opencv-contrib-python     4.10.0.84
opencv-python             4.10.0.84
opencv-python-headless    4.10.0.84
opt_einsum                3.4.0
orjson                    3.10.15
overrides                 7.4.0
packaging                 24.2
pandas                    2.2.3
pandocfilters             1.5.0
parso                     0.8.4
pathspec                  0.12.1
pexpect                   4.8.0
pillow                    11.1.0
pip                       25.0
platformdirs              3.10.0
plotly                    6.0.0
ply                       3.11
portalocker               3.0.0
prettytable               3.14.0
proglog                   0.1.10
prometheus_client         0.21.0
prompt-toolkit            3.0.43
protobuf                  4.25.6
psutil                    5.9.0
ptyprocess                0.7.0
pure-eval                 0.2.2
pycodestyle               2.12.1
pycparser                 2.21
pydantic                  2.10.6
pydantic_core             2.27.2
pydub                     0.25.1
pyflakes                  3.2.0
Pygments                  2.15.1
pyparsing                 3.2.1
PyQt5                     5.15.10
PyQt5_sip                 12.13.0
PySocks                   1.7.1
python-dateutil           2.9.0.post0
python-json-logger        3.2.1
python-multipart          0.0.20
pytorch3d                 0.7.8              /mnt/data/projects/pytorch3d
pytz                      2025.1
PyYAML                    6.0.2
pyzmq                     26.2.0
qtconsole                 5.6.1
QtPy                      2.4.1
referencing               0.30.2
regex                     2024.11.6
requests                  2.32.3
rfc3339-validator         0.1.4
rfc3986-validator         0.1.1
rich                      13.9.4
rpds-py                   0.22.3
ruff                      0.9.6
safehttpx                 0.1.6
safetensors               0.5.2
scikit-image              0.24.0
scikit-learn              1.6.1
scipy                     1.15.2
semantic-version          2.10.0
Send2Trash                1.8.2
sentencepiece             0.2.0
setuptools                75.8.0
shellingham               1.5.4
simsimd                   6.2.1
sip                       6.7.12
six                       1.16.0
sniffio                   1.3.0
sounddevice               0.5.1
soupsieve                 2.5
stack-data                0.2.0
starlette                 0.45.3
stdlibs                   2024.12.3
stringzilla               3.11.3
sympy                     1.13.1
tabulate                  0.9.0
termcolor                 2.5.0
terminado                 0.17.1
threadpoolctl             3.5.0
tifffile                  2025.1.10
timm                      0.6.13
tinycss2                  1.4.0
tokenizers                0.15.2
toml                      0.10.2
tomli                     2.0.1
tomlkit                   0.13.2
torch                     2.2.2
torchvision               0.15.2a0
tornado                   6.4.2
tqdm                      4.66.2
trailrunner               1.4.0
traitlets                 5.14.3
transformers              4.37.2
triton                    2.2.0
typer                     0.15.1
typing_extensions         4.12.2
tzdata                    2025.1
urllib3                   2.3.0
usort                     1.0.8.post1
uvicorn                   0.34.0
wcwidth                   0.2.13
webencodings              0.5.1
websocket-client          1.8.0
websockets                14.2
wheel                     0.45.1
widgetsnbextension        4.0.13
yacs                      0.1.8
zipp                      3.21.0`

**Comment by G7b9** [comment]
python: tpp.c:83: __pthread_tpp_change_priority: Assertion `new_prio == -1 || (new_prio >= fifo_min_prio && new_prio <= fifo_max_prio)' failed.
[1]    339785 IOT instruction (core dumped)  python inference.py --image_path  --driving_video_path 

return cls(
        task_info.generate_graph_config(
            enable_flow_limiting=options.running_mode
            == _RunningMode.LIVE_STREAM
        ),
        options.running_mode,
        packets_callback if options.result_callback else None,
    )
这里 cored umped
